persistence/src/main/java/org/mskcc/smile/persistence/neo4j/DbGapRepository.java

```diff
@@ -19,7 +19,7 @@ public interface DbGapRepository extends Neo4jRepository<DbGap, UUID> {
     DbGap findDbGapBySamplePrimaryId(@Param("primaryId") String primaryId);
 
     @Query("MATCH (s: Sample)-[:HAS_METADATA]->(sm: SampleMetadata {primaryId: $dbGap.primaryId}) "
-            + "MATCH (s)-[:HAS_DBGAP]->(d: DbGap) "
+            + "MERGE (s)-[:HAS_DBGAP]->(d: DbGap) "
```

Comment:
Replaced MATCH with MERGE to create the DbGap node for the given Sample in case it doesn't exist.